<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zenom</class>
 <widget class="QMainWindow" name="Zenom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zenom</string>
  </property>
  <property name="windowIcon">
   <iconset resource="zenom.qrc">
    <normaloff>:/zenom/resources/logo.png</normaloff>:/zenom/resources/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0" colspan="3">
     <widget class="Output" name="output">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="startButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="stopButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>71</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="duration"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="frequency"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="frequencyLabel">
        <property name="text">
         <string>Frequency:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="frequencyAbb">
        <property name="toolTip">
         <string>Hertz</string>
        </property>
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="durationAbb">
        <property name="toolTip">
         <string>Second</string>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_P_rojects">
     <property name="title">
      <string>Recent &amp;Projects</string>
     </property>
     <addaction name="actionClear_Recent_Projects_Menu"/>
    </widget>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionExport_as_Matlab"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_P_rojects"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionWatch"/>
    <addaction name="actionPlot"/>
    <addaction name="actionScene"/>
    <addaction name="separator"/>
    <addaction name="actionCamera"/>
    <addaction name="naoControlPanel"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Contents"/>
    <addaction name="separator"/>
    <addaction name="action_About_zenom"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Project">
   <property name="text">
    <string>&amp;Open Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionWatch">
   <property name="text">
    <string>&amp;Watch...</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
  </action>
  <action name="actionScene">
   <property name="text">
    <string>&amp;Scene</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </action>
  <action name="action_Contents">
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_About_zenom">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionExport_as_Matlab">
   <property name="text">
    <string>Export as Matlab...</string>
   </property>
  </action>
  <action name="actionClear_Recent_Projects_Menu">
   <property name="text">
    <string>Clear Menu</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="naoControlPanel">
   <property name="text">
    <string>Nao Control Panel</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Output</class>
   <extends>QPlainTextEdit</extends>
   <header>widget/output.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="zenom.qrc"/>
 </resources>
 <connections/>
</ui>
